src/scripts/modules/media/footer/footer.less

```diff
@@ -4,6 +4,8 @@
 .media-footer {
   margin: 4rem 6rem 6rem 6rem;
 
+  @media
```

Comment:
I pulled this code out from the stash from a month or so ago. This is where I left off, so no it was not intended to be there like that.